### ISSUE TITLE
bootstrap npm/yarn project

### ISSUE THREAD

**Issue by gnrlbzik** [KorolevskiTigr2/BYBYBY#1]
# Resources

- https://docs.npmjs.com/
    - https://docs.npmjs.com/cli/v8/configuring-npm/package-json (package.json)
    - **https://docs.npmjs.com/creating-a-package-json-file**
- https://yarnpkg.com/
- https://conventionalcomments.org/
- https://www.conventionalcommits.org/en/v1.0.0/
- https://www.markdownguide.org/basic-syntax/

# Description

We need to set up our project before we can start adding dependencies and work on actual code.

# Goals

1. Finish setting up repo
     ```
     …or create a new repository on the command line
     echo "# BYBYBY" >> README.md
     git init
     git add README.md
     git commit -m "first commit"
     git branch -M main
     git remote add origin git@github.com:KorolevskiTigr2/BYBYBY.git
     git push -u origin main

     git branch -M development
     git push -u origin development
     ```
3. Create new branch of `development` branch -> called `project-bootstrapping`
    - Create package.json and feel it up with appropriate information
    - Commit this code and push it out
4. Create PR so that we can merge `project-bootstrapping` branch into `development`

# TODOs
- [x] creates package.json
- [ ] create PR and merge into development
